### PR TITLE
Update1 in index.html

### PR DESCRIPTION
**Pull request by keshavgbpecdelhi**
Only style and some movement provided in web page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,4 +50,4 @@ <h3>"Talk is cheap, Show me the code."</h3>
 </div>
 
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (keshavgbpecdelhi) [inline index.html]: this left by mistake

**Comment by ekta1999** [comment]
Your PR didn't make any change in the webpage. If you feel it made some changes then add the snapshot too that will help us to review.
Thanks for showing interest. Happy Coding ;)

**Comment by ekta1999** [comment]
Try to add styling in CSS file to make the code better understandable and distributed. Don't add it in html file untile it's an inline stryling that is needed.

**Comment by keshavgbpecdelhi** [comment]
Code can be run by copying it as it is to notepad then run it on chrome!

**Pull request merged**